### DISCUSSION TITLE
The AQI should be displayed according to the corresponding country standard

### DISCUSSION THREAD

**Discussion by PumpkinJui** [breezy-weather/breezy-weather#1234] [Ideas]
### Describe your idea

According to the [Wikipedia](https://en.wikipedia.org/wiki/Air_quality_index), the AQI standard varies in different countries. Could Breezy Weather should display the AQI in the way the locals familiar with, or I can also say, according to the specific country's standard?
For example, if I add Beijing, China to my location list, its AQI will be calculated according to China's standard. When it comes to New York, America, it will be calculated as how Americans do. A general standard can be used for those who does not have a AQI standard.

It may seems complex to understand and difficult to implement, but I believe what we see in the app should be similar to what we get from weather forecasts. The weatherperson tells me the AQI is “优” (excellent), “良” (good), “轻度污染” (lightly polluted) and so on, referring to China's standard. But when I open the app, it displays completely different things, though the digital data is the same. That would make users puzzled (see #346 and #636, which described this as “doesn't correct” and “error”), and even worse, drive them to official weather apps for the AQI they are familiar with.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.6 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Our Air Quality Index works in any place in the world because it is based on WHO recommandations.

See doc: https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality

Note that we used to have other AQI implementations and it was removed because it was overly complex, a nightmare to maintain, and it was far from accurate due to specific methods used by each country. A worldwide AQI is so much better.
